src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -974,6 +977,12 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
                 // Any errors generated below here are considered Emit diagnostics 
                 // and will not be reported to callers Compilation.GetDiagnostics()
 
+                ImmutableArray<SourceSpan> dynamicAnalysisSpans = ImmutableArray<SourceSpan>.Empty;
+                if (_moduleBeingBuiltOpt.EmitOptions.EmitDynamicAnalysisData)
+                {
+                    flowAnalyzedBody = Instrumentation.InjectInstrumentation(methodSymbol, flowAnalyzedBody, methodOrdinal, compilationState, _compilation, _diagnostics, _debugDocumentProvider, out dynamicAnalysisSpans);
```

Comment:
Even the move to LowerBodyOrInitializer can be done later, I think, as long as we track this work.